docs/source/api_commands.rst

```diff
@@ -527,9 +624,27 @@ Parameters:
 get_network_info
```

Comment:
I think what you have for this one is perfect outside of needing code blocks on the return. But this is a good basis for the rest